lib/tasks/bower-install.js

```diff
@@ -21,7 +22,7 @@ module.exports = Task.extend({
 
     ui.startProgress(chalk.green('Installing browser packages via Bower'), chalk.green('.'));
 
-    var config = bowerConfig.read();
+    var config = bowerConfig.read(path.resolve('.'));
```

Comment:
Yeah that works great. I didn't knew that exists. I saw that path.resolve pattern somewhere else in the ember-cli project and adopted it.